SPECS/mariner-repos/mariner-repos.spec

```diff
@@ -12,12 +12,13 @@ Source1:        MICROSOFT-METADATA-GPG-KEY
 Source2:        mariner-official-base.repo
 Source3:        mariner-official-update.repo
 Source4:        mariner-preview.repo
+Source5:        mariner-extras.repo
```

Comment:
Are these added to all CBL-Mariner images?

Reply:
I don't think so. I checked `core-packages` spec and it only listed mariner-repos, no other subpackages.